cli/packages/graphcool-cli-core/src/commands/deploy/index.ts

```diff
@@ -359,6 +359,9 @@ https://console.graph.cool/${encodeURIComponent(info.name)}/settings/general`)
   }
 
   private async clusterSelection(): Promise<string> {
+    let localClusters = Object.keys((this.env.rc || {}).clusters as {} || {})
```

Comment:
Proposal for these 2 lines:
```js
const localClusters = (this.env.rc && this.env.rc.clusters) ?
                      Object.keys(this.env.rc.clusters) : ['local']
```

From our current code style, we don't use `if` statements without curly braces. We should specify the style guide.